src/extension.ts

```diff
@@ -1101,11 +1101,11 @@ export class ExtensionManager implements vscode.Disposable {
         return this.runCMakeCommandForAll(cmakeProject => cmakeProject.cleanConfigureWithDebugger(ConfigureTrigger.commandCleanConfigureAllWithDebugger, debuggerInformation), undefined, true);
     }
 
-    configure(folder?: vscode.WorkspaceFolder, showCommandOnly?: boolean) {
+    configure(folder?: vscode.WorkspaceFolder, sourceDir?: string, showCommandOnly?: boolean) {
```

Comment:
This technically gets exposed as a public VS Code command, that can programmatically be invoked, in which case, this change would break our current "contract". 

To fix this, please move the `sourceDir` parameter to be the last parameter, this will not break the contract.

Reply:
done